tests/run-valgrind.in

```diff
@@ -3,5 +3,6 @@
 @VALGRIND@ --error-exitcode=1 \
     --leak-check=yes \
     --show-possibly-lost=yes \
+    --errors-for-leak-kinds=definite \
```

Comment:
Was the default `possible` option causing spurious alerts?

Reply:
Oh, I spent so much time to understand why and where those possible memory leaks happens. And just now I removed "--errors-for-leak-kinds=definite" to show You the errors, and they has disappeared.

Reply:
Just pushed changes to check on CI

Reply:
Could it have been from something that one of us unintentionally fixed in the meantime between rebases? Strange.

Reply:
I think Yes. Because after I have been fixed it, I made rebase.

Reply:
CI fails, but with another memory leaks than I had before.